apis/v1alpha1/generator.yaml

```diff
@@ -38,6 +38,10 @@ resources:
         - InvalidParameterCombination
         - InvalidParameterValue
         - MissingParameter
+    fields:
+      Tags:
```

Comment:
just confirming so I can pick this change up - is this as simple as this field wont be considered by the diff checker ?

Reply:
Correct delta.go will not check for diffs in tags